is_first_sentence_with_right_order.py

```diff
@@ -0,0 +1,70 @@
+# TODO: refactor and improve code
+import csv
+import os
+import json
+import sys
+from configparser import ConfigParser
+
+'''
+Discription:
+This code computes % of first 3 sentences in extracted summaries in the right order
+
+Arguments:
+argv[1] - folder with extracted summaries
+'''
+
+file = 'config.ini'
+config = ConfigParser()
+config.optionxform = lambda option: option
+config.read(file)
+
+
+def process_text(filename):
```

Comment:
This is a utility function and is exactly the same as `process_text` in `is_first_sentence.py`. **All** such functions should be moved in a separate module (e.g. `utils.py`) and used elsewhere via import  (instead of copying each time).